services/app-api/utils/newSubmission.js

```diff
@@ -31,11 +31,14 @@ export default async function newSubmission(newData, config) {
     newData.GSI1pk = `OneMAC#${config.packageGroup}`;
     newData.GSI1sk = pk;
   } else {
-    [newData.parentId, newData.parentType] = config.getParentInfo(
-      newData.componentId
-    );
-    newData.GSI1pk = newData.parentId;
-    newData.GSI1sk = config.componentType;
+    if (newData.parentId) {
+      newData.GSI1pk = newData.parentId;
```

Comment:
yeah, if its in this branch of logic it would be  because it doesnt have a package group so I thought we should expect a parentId....but if not I also kept the old lookup way in the else using the config.getParentInfo